chainerx_cc/chainerx/context.h

```diff
@@ -61,6 +61,21 @@ class Context {
     // Gets the native backend.
     native::NativeBackend& GetNativeBackend();
 
+    // Registers the backend.
+    template <typename BackendType>
+    Backend& CreateBackend(const std::string& backend_name) {
```

Comment:
The logic duplicates with a part of `Context::GetBackend`. Why not merging them?

Reply:
Fixed.